classroom/snippets/src/main/java/AddAlias.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// [START classroom_add_alias]
```

Comment:
General note: these tags can be used in DevSite to only show snippets of code. They should probably only surround the critical sections, e.g. just the addCourseAlias method below, or maybe even just its contents. I don't think you should include boilerplate like imports.

You can also have multiple tags, e.g. [START classroom_add_alias_1] or [START some_other_tag_name]. This is handy if you have a long guide that involves multiple methods, so you can reference smaller snippets of code.

Reply:
Oh I didn't know that! I've added multiple START and END tags to encompass the whole class and the relevant bits of code.